Makefile

```diff
@@ -389,22 +389,28 @@ build-tools: goversion bin/buildhelper bin/envsubst bin/lsplatform
 build-tools-all: goversion bin/buildhelper bin/envsubst bin/lsplatform bin/catkubeletconfmap bin/watchnrtattr bin/mkginkgolabelfilter
 
 bin/buildhelper:
-	@go build -o bin/buildhelper tools/buildhelper/buildhelper.go
+	LDFLAGS="-static"
+	CGO_ENABLED=0 go build -o bin/buildhelper tools/buildhelper/buildhelper.go
```

Comment:
this is used only by the build process itself and should not be released.
Plus: how is `LDFLAGS` consumed? implicitely?

Reply:
removed unneeded tools here and below, and fixed the ldflags, which was missing in the go build cmd. thanks for catching